pkg/yang/entry_test.go

```diff
@@ -3343,3 +3344,15 @@ func TestLeafEntryTypes(t *testing.T) {
 		})
 	}
 }
+
+func TestLess(t *testing.T) {
```

Comment:
Thanks for adding a test! Can you make it more comprehensive: use table-based testing like the test above, and checking each edge case?

Reply:
Thanks for creating the cases, they look good. Could you put them into table format, like the test directly above?

This way you don't need to repeat the test logic.